Sources/Core/SourceFile.swift

```diff
@@ -28,6 +28,11 @@ public struct SourceFile {
     self.storage = storage
   }
 
+  /// Creates an instance representing the file at `filePath`.
+  public init(atPath filePath: String) throws {
```

Comment:
```suggestion
  public init(at filePath: String) throws {
```

The label might contain needless information. Looking at a use site `SourceFile(at: someBinding)`, it seems obvious to me that `someBinding` is a path.